hi-ml-histopathology/src/histopathology/datamodules/panda_module.py

```diff
@@ -32,3 +32,26 @@ def get_splits(self) -> Tuple[PandaTilesDataset, PandaTilesDataset, PandaTilesDa
         return (PandaTilesDataset(self.root_path, dataset_df=splits.train),
                 PandaTilesDataset(self.root_path, dataset_df=splits.val),
                 PandaTilesDataset(self.root_path, dataset_df=splits.test))
+
+
+class PandaSlidesDataModule(SlidesDataModule):
+    """ PandaSlidesDataModule is the child class of SlidesDataModule specific to PANDA dataset
+    Method get_splits() returns the train, val, test splits from the PANDA dataset
```

Comment:
Just to highlight that this split could bring differences with the MONAI pipeline - PANDA has some presplit in train and validation. Here we're ignoring those splits and, I understand, splitting the training dataset only in train/val/test

Reply:
Yes, I noticed that as well and that's why I have created this SubPandaDataModule in the previous PR. Do you think we should go for a fix presplit? In which case we won't be able to cross validate...

Reply:
I think it's fine for now. Just soemthing to keep in mind when we will want to compare results